packages/graph/hash_graph/lib/graph/src/api/rest/middleware.rs

```diff
@@ -84,6 +88,21 @@ where
     Ok(bytes)
 }
 
+pub fn span_trace_layer() -> TraceLayer<
+    SharedClassifier<ServerErrorsAsFailures>,
+    impl Fn(&hyper::Request<hyper::Body>) -> tracing::Span + Clone,
+    DefaultOnRequest,
+    impl Fn(&hyper::Response<axum::body::BoxBody>, std::time::Duration, &tracing::Span) + Clone,
+    DefaultOnBodyChunk,
+    DefaultOnEos,
+    impl Fn(ServerErrorsFailureClass, Duration, &tracing::Span) + Clone,
+> {
```

Comment:
These are unnecessary
```suggestion
pub fn span_trace_layer() -> TraceLayer<
    SharedClassifier<ServerErrorsAsFailures>,
    impl Fn(&Request<Body>) -> tracing::Span + Clone,
    DefaultOnRequest,
    impl Fn(&hyper::Response<axum::body::BoxBody>, Duration, &tracing::Span) + Clone,
    DefaultOnBodyChunk,
    DefaultOnEos,
    impl Fn(ServerErrorsFailureClass, Duration, &tracing::Span) + Clone,
> {
```

Reply:
Also `hyper::header::HeaderName` below on line 117 doesn't need `hyper::` but it's too far away from the diff for me to comment.

Reply:
Thanks, done in 39700b961cfd014f3048ac96564a5a8c0837c584